code/game/jobs/job/job.dm

```diff
@@ -2,69 +2,88 @@
 
 	/// The name of the job
 	var/title = "NOPE"
-	//Job access. The use of minimal_access or access is determined by a config setting: config.jobs_have_minimal_access
-	/// The access this job starts with when the server has a lower population, or if dictated by the config.
+	/// List of which accesses this job has when config.jobs_have_minimal_access is true.
 	var/list/minimal_access = list()
-	/// The access this job starts with when the server has a higher population.
+	/// List of which accesses this job has when config.jobs_have_minimal_access is false.
 	var/list/access = list()
 	/// Defines starting software that's installed on spawned tablets and laptops.
 	var/list/software_on_spawn = list()
-	/// Bitflag for every department this job is involved with.
+	/// Bitflag representing which departments this job is part of.
 	var/department_flag = 0
-	/// How many people, in total, can have this job
+	/// How many players can be this job
 	var/total_positions = 0
-	/// How many people can spawn in with this job
+	/// How many players can spawn in as this job
 	var/spawn_positions = 0
-	/// How many people currently have this job
+	/// How many players currently have this job
 	var/current_positions = 0
 
-	/// Color used to represent this job (e.g. for the selection screen)
+	/// Selection screen color
 	var/selection_color = "#515151"
-	/// List of alternative titles, if any. May be used as an associative list - the key is the title and the value is an outfit.
+	/// List of alternate titles, if any. May be an associative list - key = title, value = outfit datum.
 	var/list/alt_titles
-	var/minimal_player_age = 0				// If you have use_age_restriction_for_jobs config option enabled and the database set up, this option will add a requirement for players to be at least minimal_player_age days old. (meaning they first signed in at least that many days before.)
-	/// The main department this job is in.
+	/// If you have use_age_restriction_for_jobs config option enabled and the database set up, this option will add a requirement for players to be at least minimal_player_age days old. (meaning they first signed in at least that many days before.)
+	var/minimal_player_age = 0
+	/// Does this position have a department tag? // TODO: move to department_flag
 	var/department = null
-	/// If true, this job is counted as a head of its department
-	var/head_position = FALSE
-	var/minimum_character_age				// List of species = age, if species is not here, it's auto-pass
+	/// Is this position a head position (i.e. part of Command)?
+	var/head_position = 0
```

Comment:
FALSE is just a define for 0, so changing this to zero just decreases readability.

Reply:
Fixed